R/openxlsx2.R

```diff
@@ -85,11 +90,11 @@ openxlsx2_celltype <- c(
 )
 
 #' Deprecated Functions in Package *openxlsx2*
-#' @description
```

Comment:
Add links to deprecated functions topic, and wb_get_sheet_names

Reply:
You might know more about roxygen2, why did you remove the line? I mean it's not wrong I assume, is it just a personal preference? I do not really care, this is just for my personal education.

Reply:
No thanks, it's fine. I was just curious.